src/commands/cmd_stream.cc

```diff
@@ -358,6 +360,101 @@ class CommandXClaim : public Commander {
   }
 };
 
+class CommandAutoClaim : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+    key_name_ = GET_OR_RET(parser.TakeStr());
+    group_name_ = GET_OR_RET(parser.TakeStr());
+    consumer_name_ = GET_OR_RET(parser.TakeStr());
+    {
```

Comment:
Why?

Reply:
Hi @torwig ,

The result `s` in this block is type of `StatusOr<uint64t>` and the next block's `s` is type of `Status`.
I think they should both represent a “status” meaning and can share one variable name.
So, I chose to use a new sub block to limit the scope of `s`.

I will use two precise variable names rather than sharing same variable name with shadow style.

Best Regards,
Edward